### DISCUSSION TITLE
Prevent Expand column from growing

### DISCUSSION THREAD

**Discussion by davegomez** [KevinVandy/material-react-table#1253] [Q&A] [answered]
When I use an expanding tree structure, I don't find a way to define properties like size or grow to the "Expand" column.

My table only has one column that should grow; the remaining columns have a fixed size, but the Expand column is growing and taking space from the main column, which I don't want.

Can I specify the styling properties for the Expand column?

Because I only find information about how to customize the styles for the expansion button, not the column.

![Screenshot 2024-09-26 at 15 11 51](https://github.com/user-attachments/assets/15414daa-6e9a-4cdd-bbfb-7d449c10db67)

**Comment by KevinVandy** [comment, marked as answer]
Read the docs
https://www.material-react-table.com/docs/guides/display-columns
